optimum/exporters/openvino/model_patcher.py

```diff
@@ -1771,3 +1787,222 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._model.transformer._update_causal_mask = self._model.transformer._orig_update_causal_mask
         for block in self._model.transformer.blocks:
             block.ffn.experts.forward = block.ffn.experts._orig_forward
+
+
+def _persimmon_self_attn_sdpa_forward(
```

Comment:
I'd be in favor of adding this only for very popular architectures (as what's been done for llama / mistral) and not for all architectures as it makes things complicated for maintenance

Reply:
having sdpa is important for us from optimization point, I plan to commit it directly to transformers in near future like doing before for stablelm, then remove from patcher

Reply:
from now on let's only add this on optimum-intel for very popular models and directly add it to transformers when possible